src/Request/RequestHandler.php

```diff
@@ -184,15 +124,15 @@ private function requestToken()
      * @param string $path The path
      * @param array $parameters The request parameters
      *
-     * @return []
+     * @return array
      *
      * @throws ApiException
      */
     public function request(string $method, string $path, array $parameters = [])
     {
         $options = [
             'headers' => [
-                'Authorization' => $this->getAuthToken(),
+                'Authorization' => "Key " . $this->getApiKey(),
```

Comment:
This should be `sprintf` with single quotes, and you can remove the getApiKey method and call completely as you can reference the variable directly now that no transformation is required.

Reply:
@clone1018 I need you to please make this change so that we can merge this in and release. Thank you.